fundamentals/installation.md

```diff
@@ -45,15 +45,20 @@ If you installed napari with `pip` you can upgrade by calling
 $ pip install napari --upgrade
 ```
 
-## choosing a different backend
-By default, napari uses [PySide2](https://wiki.qt.io/Qt_for_Python). This is what most users will want. 
+## choosing different Qt backends
 
-If you want to use [PyQt](https://www.riverbankcomputing.com/software/pyqt/intro) instead for your backend, you can run these lines after installing napari:
+Most users can safely ignore this section. If, however, you aim to use napari together with software that uses PyQt5, or have some other reason to prefer that Qt backend, read on.
+
+[Qt](https://www.qt.io) is a C++ library that provides unified graphical user interface (GUI) elements and interactivity across platforms and operating systems. Several libraries exist for accessing these elements from Python, which we call in the context of napari a *backend*.
+
+The default napari backend is [PySide2](https://wiki.qt.io/Qt_for_Python), and this is what you get if you type `pip install napari` or `conda install -c conda-forge napari` and you do not have [pyqt5](https://pypi.org/project/PyQt5/) installed. If you wish to use [PyQt5](https://www.riverbankcomputing.com/software/pyqt/intro) as the backend instead, you can install napari like this:
 ```
-pip uninstall pyside2 -y
 pip install pyqt5
+pip install napari
 ```
 
+**Note**: the order of the packages in the above line is important, because [pyqt5](https://pypi.org/project/PyQt5/) must be installed *before* napari.
+
```

Comment:
Ok. I found two more important informations and I suggest to add additional note. This come from using `pyproject.toml` as configuration file for black. When pip find this file it enables build package build in isolate mode (PEP 517) so code in `setup.py` can not check which packages are already installed. 

This problem does not occur when napari is installed from pypi because `pyproject.toml` is not listed in `MANIFEST.in` 

```markdown
**Note 2** If you would like install napari from repository but with PyQt5 instead of PySide2 then after install PyQt5 install napari with:

`pip install --no-use-pep517 git+https://github.com/napari/napari`

or 

`pip install --no-use-pep517 path_to_napari`


```

Reply:
This genuinely makes it seem that the "simplification" for installing napari & pyqt5/pyside2 has actually made things much more complex. @jni does this change your mind?

Reply:
And thank you @Czaki you've been very diligent with this review and it's been very, very helpful to know about all of these tricky edge cases!

Reply:
Old instruction still works, but if you install napari in editable mode without `--no-use-pep517` then remove PySide2 to use PyQt5 then you got so pretty message:

```
napari
Traceback (most recent call last):
  File "/home/czaki/.pyenv/versions/napari-test/bin/napari", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/home/czaki/.pyenv/versions/3.7.2/envs/napari-test/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3258, in <module>
    @_call_aside
  File "/home/czaki/.pyenv/versions/3.7.2/envs/napari-test/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3242, in _call_aside
    f(*args, **kwargs)
  File "/home/czaki/.pyenv/versions/3.7.2/envs/napari-test/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3271, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/czaki/.pyenv/versions/3.7.2/envs/napari-test/lib/python3.7/site-packages/pkg_resources/__init__.py", line 584, in _build_master
    ws.require(__requires__)
  File "/home/czaki/.pyenv/versions/3.7.2/envs/napari-test/lib/python3.7/site-packages/pkg_resources/__init__.py", line 901, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/czaki/.pyenv/versions/3.7.2/envs/napari-test/lib/python3.7/site-packages/pkg_resources/__init__.py", line 787, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'PySide2>=5.12.3' distribution was not found and is required by napari
```